apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/ShardInitTableSchemaVersion.java

```diff
@@ -15,7 +15,7 @@ protected int update(int nextUpdate) {
                         "DB_ID BIGINT not null, " +
                         "ID BIGINT not null, " +
                         "VERSION INTEGER not null, " +
-                        "TIMESTAMP INTEGER not null, " +
+                        "\"TIMESTAMP\" INTEGER not null, " +
```

Comment:
Same field is also present in AplDbVersion

Reply:
no need to do it there

Reply:
done